includes/controls/groups/grid-container.php

```diff
@@ -33,7 +33,7 @@ protected function init_fields() {
 			'label_on' => esc_html__( 'Show', 'elementor' ),
 			'label_off' => esc_html__( 'Hide', 'elementor' ),
 			'default' => 'yes',
-			'frontend_available' => true,
+			'editor_available' => true,
```

Comment:
Changing the key to `editor_available` will make sure the settings will not render on the FE of the website, but still be available in the Editor.